src/main/webapp/TempInfoWindow.js

```diff
@@ -60,72 +60,38 @@ class TempInfoWindow extends InfoWindowTemplate {
    * Returns the HTML of the right section of the information window
    */
   makeRightColumn_() {
-    let rightCol = document.createElement("div");
-    rightCol.classList.add(InfoWindowTemplate.RIGHT_COLUMN);
-    rightCol.appendChild(this.makeContentForm_());
-    return rightCol;
-  }
-
-  /**
-   * @Private
-   * Returns the HTML for the input form for the marker description
-   */
-  makeContentForm_() {
-    let form = document.createElement("div");
+    let rightCol = myMap.makeEl("div", InfoWindowTemplate.RIGHT_COLUMN);
 
-    //get the marker that is currently being edited to populate the form
     let editedMarker = myMap.editingPermMarker();
+    let titleText = editedMarker? editedMarker.getTitle(): null;
+    let bodyText = editedMarker? editedMarker.getBody(): null;
 
-    form.appendChild(this.makeTitleInput_(editedMarker));
-    form.appendChild(this.makeBodyInput_(editedMarker));
-    form.appendChild(this.makeSubmitBtn_());
-    return form;
-  }
-
-  /**
-   * @Private
-   * Builds the HTML for the textarea to input the title for the marker
-   * @param {PermMarker} editedMarker perm marker that is being edited
-   */
-  makeTitleInput_(editedMarker) {
-    let titleInput = document.createElement("textarea");
-    titleInput.id = TempInfoWindow.TITLE_INPUT;
-    titleInput.placeholder = TempInfoWindow.DEFAULT_TITLE;
+    let titleInput = this.makeForm_(TempInfoWindow.TITLE_INPUT,
+        TempInfoWindow.DEFAULT_TITLE, titleText);
```

Comment:
So, this titleText is used as ID? I'm wondering it is possible that 2 title Texts to be exactly the same, which resulting in identical ids?

Reply:
+1. Do we need an ID here?

Reply:
In this context, `titleText` represents the innerHTML of the titleInput element, `TempInfoWindow.DEFAULT_TITLE` is the element's placeholder, and `TempInfoWindow.TITLE_INPUT` is the element's id (the parameters of `makeForm_()` are slightly different than that of `makeEl()`). Since there is only one instance of TempInfoWindow per user, it is impossible for there to exist more than one `titleInput` object, and therefore impossible for there to be two `titleInput` objects with the same ID.

Reply:
The reason I need an ID is that I can only set the submit event listener for the tempInfoWindow form after it has been added to the DOM, so the easiest way to access it is through an ID.